src/components/TimeEntries.tsx

```diff
@@ -44,7 +44,7 @@ function TimeEntries() {
 
   useEffect(() => {
     const newTimeEntries = [];
-    rawTimeEntries.map((timeEntry) =>
+    rawTimeEntries.forEach((timeEntry) =>
       newTimeEntries.push({
         id: timeEntry.id,
         client: timeEntry.client,
```

Comment:
Using `push` is something that you'll want to avoid if possible. You can use a the `map` method here to create a custom array. For example:
```
const timeEntry = timeEntries.map( entry => {
  return {
    id: timeEntry.id,
  }
}
```